nautobot/docs/development/getting-started.md

```diff
@@ -246,6 +246,9 @@ After opening the project directory in VS Code in a supported environment, you w
 
 To start Nautobot, select **Run Without Debugging** or **Start Debugging** from the Run menu. Once Nautobot has started, you will be prompted to open a browser to connect to Nautobot.
 
+!!! note
+    You can run tests with `nautobot-server test --config=nautobot/core/tests/nautobot_config.py` while inside the Container.
```

Comment:
Would you mind also updating the `nautobot-server test` entries on lines 430 and 436 below?

Reply:
Maybe this isn't strictly necessary (since someone using the virtualenv may not have the rqworker running at all) but I figure better consistent than not.

Reply:
Consistent works for me.  I updated the docs.